lib/rubocop/cop/lint/ensure_return.rb

```diff
@@ -3,6 +3,9 @@
 module RuboCop
   module Cop
     module Lint
+      # Explicit return from an ensure block alters the control flow
```

Comment:
I think the extra paragraph should come after the high-level description, not before it.

Reply:
Done.